api/Document.json

```diff
@@ -6951,7 +6951,10 @@
         "__compat": {
           "description": "<code>scroll</code> event",
           "mdn_url": "https://developer.mozilla.org/docs/Web/API/Document/scroll_event",
-          "spec_url": "https://drafts.csswg.org/cssom-view/#eventdef-document-scroll",
+          "spec_url": [
+            "https://drafts.csswg.org/cssom-view/#eventdef-document-scroll",
```

Comment:
Can you update the `Element` entry with the same URLs? This one is actually the one for elements too, despite "document" in the fragment.